code/parse/sexp.cpp

```diff
@@ -38875,31 +38880,37 @@ SCP_vector<sexp_help_struct> Sexp_help = {
 	{ OP_PERCENT_SHIPS_DEPARTED, "percent-ships-departed\r\n"
 		"\tBoolean function which returns true if the percentage of ships in the listed ships and wings "
 		"which have departed is greater or equal to the given percentage.  For wings, all ships of all waves "
-		"are used for calculation for the total possible ships to depart.\r\n\r\n"
+		"are used for calculation for the total possible ships to depart.  As expected, ships yet to arrive are "
```

Comment:
"As expected," is not necessarily true.  Some users may have the expectation that it may follow the pattern of only evaluating ships that have already arrived from other sexps.  The documentation is better without "As expected".